experimental/segmenter/data/word.toml

```diff
@@ -0,0 +1,705 @@
+segmenter_type = "word"
+
+[[tables]]
+name = "CR"
+
+[[tables]]
+name = "LF"
+
+[[tables]]
+name = "Newline"
+
+[[tables]]
+name = "Extend"
+
+[[tables]]
+name = "ZWJ"
+
+[[tables]]
+name = "Regional_Indicator"
+
+[[tables]]
+name = "Format"
+
+[[tables]]
+name = "Katakana"
+
+[[tables]]
+name = "Hebrew_Letter"
+
+[[tables]]
+name = "ALetter"
+
+[[tables]]
+name = "Single_Quote"
+
+[[tables]]
+name = "Double_Quote"
+
+[[tables]]
+name = "MidNumLet"
+
+[[tables]]
+name = "MidLetter"
+
+[[tables]]
+name = "MidNum"
+
+[[tables]]
+name = "Numeric"
+
+[[tables]]
+name = "ExtendNumLet"
+
+[[tables]]
+name = "WSegSpace"
+
+[[tables]]
+name = "Extended_Pictographic"
+
+[[tables]]
+name = "SA"
+
+[[tables]]
+name = "ALetter"
+left = "ALetter"
+right = "Extend"
```

Comment:
Question (non-blocking): Is ``[[table]]`` with `name == left` used to implement [WB4](https://www.unicode.org/reports/tr29/tr29-39.html#WB4) rule? That is, is it correct that the state should remain in "ALetter" if we see "ALetter" followed by "Extend"?

Reply:
Yes, previous rules (WB1-WB3) don't have any ALetter rules, so WB4 should be applied.